res/css/structures/_CreateRoom.scss

```diff
@@ -25,7 +25,7 @@ limitations under the License.
 .mx_CreateRoom textarea {
     border-radius: 3px;
     border: 1px solid $strong-input-border-color;
-    font-weight: 300;
+    font-weight: 400;
```

Comment:
As per https://github.com/matrix-org/matrix-react-sdk/pull/7880#issuecomment-1049808853, this entire file is unused, and so it should be deleted.

Reply:
> As per [#7880 (comment)](https://github.com/matrix-org/matrix-react-sdk/pull/7880#issuecomment-1049808853), this entire file is unused, and so it should be deleted.

I have removed the file as a part of this PR. Kindly have a look. 
Thanks for your review.